src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -3161,6 +3176,14 @@ bool SvgLoader::header()
         }
 
         preserveAspect = loaderData.doc->node.doc.preserveAspect;
+
+        if ((loaderData.doc->node.doc.bboxFlag & BBoxFlag::All) == BBoxFlag::All ||
+            (loaderData.doc->node.doc.bboxFlag & BBoxFlag::Viewbox) == BBoxFlag::Viewbox ||
+            (loaderData.doc->node.doc.bboxFlag & BBoxFlag::WidthHeight) == BBoxFlag::WidthHeight) {
```

Comment:
I haven't read all the svg specs, but I think we should consider the case where only one of width or height is declared.

```html
<?xml version='1.0' encoding='utf8'?>
<svg xmlns="http://www.w3.org/2000/svg" id="glyph310" width="64"><g transform="translate(0 -9.6) translate(-0.0 -104.8576) scale(2.048)"><path d="M31.4 4.9c1.1 2.5-.5 10.9-.5 10.9s-7.5-4.4-8.6-6.9c-1.1-2.5 0-5.4 2.5-6.5 2.5-1.1 5.5 0 6.6 2.5" fill="#64892f" /><path d="M33.3 16.3h-2.6c0-4.3 2.4-11.6 12-11.6v2.6c-8.7 0-9.4 8.2-9.4 9" fill="#947151" /><path d="M32 15.8C24.7 8.4 3 8.1 3 29.3 3 42.7 18.8 62 25.7 62c3.1 0 4.2-2 6.1-2 1.8 0 2.4 1.9 6.4 1.9C45.1 62 61 42.8 61 29.3 61 8.1 39.3 8.4 32 15.8z" fill="#8cc63e" /></g></svg>
```
(tested in chrome)

I don't think `BBoxFlag::WidthHeight` or `BBoxFlag::Viewbox` are necessary in that sense.

Reply:
yes, you are right, I'll change it

Reply:
done in #1318